android/app/src/main/java/org/chrome/device/ml/experiments/Experiment.java

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2020. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+package org.chrome.device.ml.experiments;
+
+/**
+ * Experiment Interface
+ */
+public interface Experiment {
+
+    // Runs initialization tasks of experiment
+    public void initialize();
+
+    // Close tasks and removes object of experiment
```

Comment:
not sure what this means based on just this interface - i wouldnt extract this out to its own interface yet until you have another type (other than MobileBertExperiment)

Reply:
I had this initially because we potentially we wanted to have different experiments. I'll remove it since we only have one experiment.

Reply:
Actually is it okay if we keep it? Because it would be easier if we ever decided to add other experiments.